stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -790,10 +808,10 @@ public class ChannelController internal constructor(
      * If the request fails we retry according to the retry policy set on the repo
      */
     internal suspend fun sendReaction(reaction: Reaction, enforceUnique: Boolean): Result<Reaction> {
-        val currentUser = domainImpl.currentUser
+        val currentUser = domainImpl.user.value
         reaction.apply {
             user = currentUser
-            userId = currentUser.id
+            userId = currentUser?.id ?: "user_not_set"
```

Comment:
Why does it make sense to use this dummy ID here? Shouldn't we just fail the operation in some way if we don't have a user?

Additionally: what if a user's ID is `"user_not_set"`? I'd be very careful about using something like this anywhere in the SDK.

Reply:
Removed now.